### ISSUE TITLE
Error: Formulary is not a class

### ISSUE THREAD

**Issue by ilovezfs** [youtux/homebrew-livecheck#13]
Looks like https://github.com/Homebrew/brew/pull/2066 broke `brew livecheck`:

```
Error: Formulary is not a class
/usr/local/Homebrew/Library/Taps/youtux/homebrew-livecheck/livecheck/extend/formulary.rb:3:in `<top (required)>'
/usr/local/Homebrew/Library/Homebrew/vendor/portable-ruby/2.0.0-p648/lib/ruby/2.0.0/rubygems/core_ext/kernel_require.rb:55:in `require'
/usr/local/Homebrew/Library/Homebrew/vendor/portable-ruby/2.0.0-p648/lib/ruby/2.0.0/rubygems/core_ext/kernel_require.rb:55:in `require'
/usr/local/Homebrew/Library/Taps/youtux/homebrew-livecheck/cmd/brew-livecheck.rb:7:in `<top (required)>'
/usr/local/Homebrew/Library/Homebrew/vendor/portable-ruby/2.0.0-p648/lib/ruby/2.0.0/rubygems/core_ext/kernel_require.rb:55:in `require'
/usr/local/Homebrew/Library/Homebrew/vendor/portable-ruby/2.0.0-p648/lib/ruby/2.0.0/rubygems/core_ext/kernel_require.rb:55:in `require'
/usr/local/Homebrew/Library/Homebrew/brew.rb:24:in `require?'
/usr/local/Homebrew/Library/Homebrew/brew.rb:100:in `<main>'
```

**Comment by rwhogg** [comment]
Can this issue can be closed now that #14 was merged?

**Comment by youtux** [comment]
Fixed by #14
